docs/docs/configuration/index.md

```diff
@@ -477,6 +477,18 @@ cameras:
       order: 0
       # Optional: Whether or not to show the camera in the Frigate UI (default: shown below)
       dashboard: True
+      # Optional: Override browser locale and show time in 12 / 24 hour format (default: use browser locale)
+      use12hour: False
```

Comment:
Looking at this again, I think it makes more sense to use an enum here called `time_format` with options `12-hour`, `24-hour`, `browser`.